bench/workgen/runner/chunkcache_simple.py

```diff
@@ -41,12 +41,21 @@ def get_stat(session, stat):
     stat_cursor.close()
     return val
 
-# Set up the WiredTiger connection.
+def tiered_config(home):
+    bucket_path = home + "/" + "bucket2"
+    if not os.path.isdir(bucket_path):
+        os.mkdir(bucket_path)
+
+# Set up the WiredTiger connection
 context = Context()
-dir = os.getcwd()
-conn_config = 'create,statistics=(all),statistics_log=(wait=1,json=true,on_close=true)'
-chunkcache_config = conn_config + f',chunk_cache=[enabled=true,chunk_size=10MB,capacity=1GB,type=FILE,storage_path={dir}/chunkcache_tmp]'
-conn = context.wiredtiger_open(conn_config)
+wt_builddir = os.getenv('WT_BUILDDIR')
+if not wt_builddir:
+    wt_builddir = os.getcwd()
+conn_config = f'create,statistics=(all),statistics_log=(wait=1,json=true,on_close=true),\
+    tiered_storage=(auth_token=Secret,bucket=bucket2,bucket_prefix=pfx_,name=dir_store),\
+        extensions=({wt_builddir}/ext/storage_sources/dir_store/libwiredtiger_dir_store.so=(early_load=true))'
+chunkcache_config =  conn_config + f',chunk_cache=[enabled=true,chunk_size=10MB,capacity=1GB,type=FILE,storage_path={wt_builddir}/chunkcache_simple]'
```

Comment:
Hey @ruby-oujo could we change the storage_path to the following -

`storage_path=/tmp/chunkcache_simple`

This is because, in case of a failure, we do not wish to upload this file to evergreen as it could be very large.

Reply:
dome